rocketpy/mathutils/function.py

```diff
@@ -2229,7 +2229,7 @@ def integral(self, a, b, numerical=False):
             ans = np.trapz(y_integration_data, x_integration_data)
         else:
             # Integrate numerically
-            ans, _ = integrate.quad(self, a, b, epsabs=0.001, limit=10000)
+            ans, _ = integrate.quad(self, a, b, epsabs=1e-4, epsrel=1e-3, limit=1000)
```

Comment:
This is, likely, the most polemic line of this PR. Therefore, I will explain the main points that motivated this change:

- According to the [scipy quad docs](https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.quad.html) the tolerances follow: `abs(i-result) <= max(epsabs, epsrel*abs(i))`;

- The integration warning on the tests happened in the computation of the average of the exhaust velocity. The curve was fairly well behaved, but likely had some property that made the integrator overstimate the error to about 10 (while the result was about `50_000`).

- Then, the configs prior to this PR were `epsabs=1e-3` and `epsrel=1.49e-8`. It is easy to see that the relative tolerance is this case is basically meaningless.

In this PR, I have set up the `epsabs=1e-4` and `epsrel=1e-3` so that the integral error is either smaller than before (`epsabs` reduction) or, at least, three significant digits smaller than the integral value. 

I believe that this allows for a better balance between precision and steps taken than before while having an `epsrel` more meaningful, insofar as integrals that involve really large values do not require the tight precision dictated by `absrel`.

Nonetheless, I would like to hear your opinions on the matter. If this comment did not made the issue clear enough, please do not hesitate in telling me to reword it.

Reply:
@phmbressan , I understood your situation and I think this is totally fine.

As a final user, I don't see any differences in the code  if you chose one kind of error or another (abs/rel).